README.md

```diff
@@ -189,10 +185,39 @@ otherwise.
 
 ## Practicing What We've Learned
 
-The REPL below contains a number of expressions that use the logical operators
+The code below contains a number of expressions that use the logical operators
 we've learned about in this lesson:
 
-<iframe height="400px" width="100%" src="https://repl.it/@LizBurton/EmbellishedPresentOrigin?lite=true" scrolling="no" frameborder="no" allowtransparency="true" allowfullscreen="true" sandbox="allow-forms allow-pointer-lock allow-popups allow-same-origin allow-scripts allow-modals"></iframe>
+```js
+//What should each of the following expressions return? Once you have an answer, copy the expression in question into the console and hit enter to see if you're right!
+
+// Practice with AND
+0 && false;
+
+// 0 && true;
+
+// true && NaN;
+
+// true && !1;
+
+// !0 && "This is a string";
+
+// !0 && "";
+
+// !0 && !!"";
+
+// Practice with OR
+
+// 0 || false;
+
+// true || false;
+
+// true || 1;
+
+// !true || !false
+
+// !1 || !0
+```
 
```

Comment:
There's no real reason to comment out the code lines here now that we're not using the repl. I think it'll be easier for students to read and understand if just the actual comments are commented out. You'll need to update the instructions to match.

We also need to edit the "A Note About Comments" section. I suggest changing the first sentence to "Note that some of the lines in the code above currently...", adding a sentence to the end of the first paragraph: "You'll learn more about this later.", and removing the second paragraph. 

Unfortunately, without the repl, there's no way to illustrate the use of comments to keep code from executing. We'll need to find a place to cover that later in the curriculum.